File: beginner/pagina02/1043/1043.ts
const input = require('fs').readFileSync("./test/stdin", 'utf8');
const lines = input.split(' ');

const A = Number(lines.shift());
const B = Number(lines.shift());
const C = Number(lines.shift());

const perimetro = (A + B + C)
const area = ((A + B) * C ) / 2;
const condicao = A < (B + C) && B < ( A + C) && C < (A + B)

console.log( condicao ? `perimetro = ${perimetro.toFixed(1)}` : `Area = ${area.toFixed(1)}`)

File: beginner/pagina02/1043/1043.cpp
#include <iostream>
#include <iomanip>

using namespace std;

int main(){

    float A, B, C;
    cin >> A >> B >> C;

    float perimetro = (A + B + C);
    float area = ((A + B) * C ) / 2;
    bool condicao = A < (B + C) && B < ( A + C) && C < (A + B);

    if(condicao){
        cout << "Perimetro = " <<  fixed << setprecision(1) << perimetro << endl;
    } else {
        cout << "Area = " << fixed << setprecision(1) << area << endl;
    }

    return 0;
}